### ISSUE TITLE
Compile method on Model doesn't accept Dictionary for `loss` parameter

### ISSUE THREAD

**Issue by Engid** [SciSharp/Keras.NET#19]
Greetings! First I would like to say thank you for providing the world with Keras.NET! 

I am working on translating the [model](https://github.com/AppliedDataSciencePartners/DeepReinforcementLearning/blob/master/model.py) from this [AlphaZero tutorial](https://medium.com/applied-data-science/how-to-build-your-own-alphazero-ai-using-python-and-keras-7f664945c188), and I think I've hit a snag in translating the [loss parameter here](https://github.com/AppliedDataSciencePartners/DeepReinforcementLearning/blob/1e3b19a28c4549e8ca3cda41c05dbdfbbf436c4b/model.py#L235). The author is using a dictionary to specify two loss functions, one for the policy head and one for the value head (as well as two [`loss_weights`](https://github.com/AppliedDataSciencePartners/DeepReinforcementLearning/blob/1e3b19a28c4549e8ca3cda41c05dbdfbbf436c4b/model.py#L237)). It looks like there is a `PyDict` type defined for a python dictionary in the `Python.Runtime` assembly, but it's not implemented in the [`ToPython`](https://github.com/SciSharp/Keras.NET/blob/b67cffa10ea6f4bf25277e30be2ebef4a2dbafcc/Keras/Keras.cs#L68) method within the `Keras` class. 

I could probably work on a PR for this change, but I am pretty new to the `Python.Runtime` assembly, and to the `Keras.NET` library. If someone could help me through the change, I don't think it would be too hard. (Also, if someone with more experience wanted to handle it, then I would greatly appreciate it!)

Thank you,
-Nick

**Comment by deepakkumar1984** [comment]
Update the code with overload for compile method. If your model has 2 outputs then you can pass 2 loss function. Please take the latest code and test it out. Will be released to nuget with some more fixes

**Comment by Engid** [comment]
Sounds good, thank you. I will send a PR soon.
